File: backEnd/src/main/java/com/quiz/ourclass/domain/relay/dto/response/SendRelayResponse.java
package com.quiz.ourclass.domain.relay.dto.response;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Builder;

@Builder
@Schema(description = "이어달리기 전달 응답 DTO")
public record SendRelayResponse(
    @Schema(description = "이어달리기 이전 주자 이름")
    String prevMemberName,
    @Schema(description = "이어달리기 이전 주자가 받은 질문")
    String prevQuestion
) {

}

File: android/data/src/main/java/com/sixkids/data/model/response/SendRelayResponse.kt
package com.sixkids.data.model.response

import com.sixkids.model.RelaySend

data class SendRelayResponse(
    val prevMemberName: String,
    val prevQuestion: String
)

internal fun SendRelayResponse.toModel() = RelaySend(
    prevMemberName = prevMemberName,
    prevQuestion = prevQuestion
)